src/metaphlan.jl

```diff
@@ -43,13 +43,19 @@ end
 
 
 """
-Option1: take a path to merged table (eg test/files/metaphlan_multi_profile.tsv)
+Option1: take a path to merged table (eg test/files/metaphlan_multi_test.tsv)
     and make CommunityProfile
 
-Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1_profile.tsv", "test/files/metaphlan_single2_profile.tsv"])
+Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1.tsv", "test/files/metaphlan_single2.tsv"])
     and make CommunityProfile
 """
-function metaphlan_profiles(tables)
+function metaphlan_profiles(path::AbstractString; level=:all)
+    profiles = CSV.read(path, DataFrame)
+    taxa = [last(_split_clades(c)) for c in profiles[:, "#SampleID"]]
+    mat = sparse(Matrix(profiles[:, 2:end]))
+    samples = MicrobiomeSample.(first.(split.(names(profiles[:, 2:end]), "_")))
```

Comment:
I don't think we want to split like this. I often have samples like `C00001_1F_1A`, and this would eliminate a bunch of that. I do think it would be reasonable to trim `_profile` off the end, if it exists, something like `replace.(names(profiles[:, 2:end], Ref("_profile" => ""))`